### PR TITLE
feat: some css changes

### PR DESCRIPTION
**Pull request by lendai**

### CODE REVIEW

File: src/components/Content/Heading.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ import HeadingLink from "../Link/HeadingLink";
 import { catalogShape } from "../../CatalogPropTypes";
 import { heading } from "../../styles/typography";
 import { css } from "../../emotion";
+import { reduce } from "rxjs/operators";
```
Comment (lendai) [inline src/components/Content/Heading.js]: is this used anywhere?

File: src/components/Page/PageHeader.js [hunk 2]
```diff
@@ -2,6 +2,7 @@ import React, { Component } from "react";
 import PropTypes from "prop-types";
 import { css } from "../../emotion";
 import { heading } from "../../styles/typography";
+import { reduce } from "rxjs/operators";
```
Comment (lendai) [inline src/components/Page/PageHeader.js]: is this used anywhere?

File: package.json [hunk 3]
```diff
@@ -80,6 +80,7 @@
     "react-document-title": "^2.0.3",
     "react-router": "^3.2.0",
     "react-router-scroll": "^0.4.2",
+    "rxjs-compat": "^6.4.0",
```
Comment (lendai) [inline package.json]: is this used anywhere?

Comment (elekarlsen) [reply package.json]: that was my attempt to fix the error in the terminal

Comment (elekarlsen) [reply package.json]: quite unsuccessful though, so I bet it's better to remove those

**Pull request merged**